File: herencia/coworking-csharp/Course.OOP.Inheritance.Business/Media/VideoBeam.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Course.OOP.Inheritance.Business.Media
{
    public class VideoBeam : ISchedulable
    {

        public string Resolution { get; set; }

        public VideoBeam(String resolution)
        {
            this.Resolution = resolution;
        }

        public bool IsAvailable(DateTime start, DateTime end)
        {
            //Implementar lógica para definir disponibilidad.
            return true;
        }
    }
}

File: herencia/coworking-java/src/main/java/course/oop/inheritance/media/VideoBeam.java
package course.oop.inheritance.media;

import course.oop.inheritance.Schedulable;

import java.time.LocalDateTime;

public class VideoBeam implements Schedulable {

    private String resolution;

    public VideoBeam(String resolution) {
        this.resolution = resolution;
    }

    public boolean isAvailable(LocalDateTime start, LocalDateTime end) {
        //Implementar lógica para definir disponibilidad.
        return true;
    }

    public String getResolution() {
        return resolution;
    }

    public void setResolution(String resolution) {
        this.resolution = resolution;
    }


}
